lib/project_types/extension/tasks/configure_features.rb

```diff
@@ -42,10 +42,12 @@ def surface_area_configurations
             renderer_package_name: "@shopify/argo-admin",
             required_fields: [:shop, :api_key],
             required_shop_beta_flags: [:argo_admin_beta],
+            cli_package_name: "@shopify/argo-admin-cli",
```

Comment:
Ideally, this would live on the Specification Handler for this specification, but since ArgoAdmin doesn't currently have a SpecificationHandler (it's defaulting to `default`), this is the best place I could think of to store this.

As cleanup, I would like to create either a `Product Subscription` or Argo specific-handler (like the one we have for [checkout](https://github.com/Shopify/shopify-app-cli/blob/master/lib/project_types/extension/models/specification_handlers/checkout_post_purchase.rb)) where some of this data could get abstracted to.